src/middleware/src/storage/src/websql.js

```diff
@@ -7,20 +7,20 @@ import isFunction from 'lodash/isFunction';
 import isString from 'lodash/isString';
 const idAttribute = process.env.KINVEY_ID_ATTRIBUTE || '_id';
 const masterCollectionName = 'sqlite_master';
-const size = 5 * 1000 * 1000; // Database size in bytes
 let dbCache = {};
-let isSupported = undefined;
+let isSupported;
 
 export default class WebSQL {
-  constructor(name = 'kinvey') {
+  constructor(name = 'kinvey', size = 5000) {
     this.name = name;
+    this.size = size;
   }
 
   openDatabase() {
     let db = dbCache[this.name];
 
     if (!db) {
-      db = global.openDatabase(this.name, 1, '', size);
+      db = global.openDatabase(this.name, 1, '', this.size);
```

Comment:
The third parameter here is used by the browser as the database name in the prompt to the user. I'm wondering what name makes sense there, but leaving it blank looks very odd (check the screenshot in MLIBZ-1593)